### DISCUSSION TITLE
npm install problem over vue

### DISCUSSION THREAD

**Discussion by olkotz** [nolimits4web/swiper#4485] [Q&A]
I tried to install swiper on my quasar school proyect and i got this error:

swiper@6.5.8 postinstall: `node postinstall.js`

Can anyone help me?

Thanks!!!!

**Comment by vltansky** [comment]
There is no error here? Sure it's not working for you?

**Reply by olkotz** [reply]
Exactly, the instalation throwme that error, and when i tried to import on my quasar project and when i try compile it torw me this errors:

 ERROR  Failed to compile with 5 errors                                                                                                                                                    11:47:11

These dependencies were not found:

* swiper/css/swiper.css in ./src/boot/slider.js
* swiper/swiper-bundle.css in ./src/boot/slider.js
* swiper/swiper.scss in ./node_modules/@quasar/app/lib/webpack/loader.transform-quasar-imports.js!./node_modules/babel-loader/lib??ref--2-0!./node_modules/@quasar/app/lib/webpack/loader.auto-import-client.js?kebab!./node_modules/vue-loader/lib??vue-loader-options!./src/pages/Profile.vue?vue&type=script&lang=js&
* swiper/vue in ./node_modules/@quasar/app/lib/webpack/loader.transform-quasar-imports.js!./node_modules/babel-loader/lib??ref--2-0!./node_modules/@quasar/app/lib/webpack/loader.auto-import-client.js?kebab!./node_modules/vue-loader/lib??vue-loader-options!./src/pages/Profile.vue?vue&type=script&lang=js&
* vue-awesome-swiper in ./src/boot/slider.js

**Reply by vltansky** [reply]
What is the error when you are running `npm install`?

**Reply by olkotz** [reply]
the instalation throw me this error:

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! swiper@6.5.8 postinstall: `node postinstall.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the swiper@6.5.8 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

**Reply by vltansky** [reply]
try `npm install --ignore-scripts` or `yarn`

**Reply by olkotz** [reply]
Yeah, this solves the problem with the instalation.... but the problems will continue, i give up with this component, dosent work right with vue and quasar. I try with another one.

Thanks for your help!
